### ISSUE TITLE
Confusion around normalising the bounding boxes between (0, 100) for fine-tuning.

### ISSUE THREAD

**Issue by Sid2697** [Vision-CAIR/MiniGPT-4#460]
Hello team,

The paper mentions that "Coordinates for X and Y are represented by integer values normalized in the range [0, 100]." Upon having a look at the code,

here:
https://github.com/Vision-CAIR/MiniGPT-4/blob/1d6b6e34f6277307d40881fd71148934afa368b8/minigpt4/datasets/datasets/vg_dataset.py#L48-L75

and here:
https://github.com/Vision-CAIR/MiniGPT-4/blob/1d6b6e34f6277307d40881fd71148934afa368b8/minigpt4/datasets/datasets/coco_dataset.py#L49-L80

If I understand correctly, the size of the processed image (obtained from `image = self.vis_processor(image)` has been ignored and a new size (`image_new_size = [100, 100]`) has been hardcoded. This theoretically makes the _dummy_ image  of size 100x100 which later helps in normalising the bounding boxes to values (0, 100).

However, as the original image's shape is different (448, 448 (according to the paper and code)), these bounding boxes fail to map correctly to the referred objects.

Shouldn't the bounding boxes be normalised considering the size of the input image? How does the training work if the mapping is not 1-1?

**Comment by edcand3ball** [comment]
Hello, I am also confused about this question. Have you found the answer
